src/views/OtherTools.md

```diff
@@ -74,7 +74,42 @@
             here</a>.
             </p>
     </div>
-</div>
+    <div class="card">
+            <h4>Phenogrid</h4>
+            <p><a href="https://github.com/monarch-initiative/phenogrid">Phenogrid</a> is a Javascript component that
+            visualizes semantic similarity calculations provided by OWLSim,
+            as provided through APIs from the Monarch Initiative. Given an input list of phenotypes and parameters 
+            specified in config/phenogrid_config.js indicating desired source of matching models (humans, model
+            organisms, etc.), the phenogrid will call the Monarch API to get OWLSim results and render them in your web
+            browser in data visualization.</p>
+    </div>
+    <div class="card">
+            <h4>Dipper</h4>
+            <p><a href="https://github.com/monarch-initiative/dipper">Dipper</a> is a Python package to generate RDF 
+            triples from common scientific resources. Dipper includes subpackages and modules to create graphical models
+            of this data. Data generated from this pipeline can be used in a variety of ways downstream. We recommend
+            loading the data into a triple store or graph database that is optimized for use with ontologies, such as
+            BlazeGraph. We also maintain SciGraph, an application that loads RDF and OWL into Neo4J. Smaller files can
+            be loaded into an ontology editor like Protege.</p>
+    </div>
+    <div class="card">
+            <h4>HPO Case Annotator</h4>
+            <p>
+            <a href="https://github.com/monarch-initiative/HpoCaseAnnotator">The HPO Case Annotator (HCA)</a> GUI is a
+            Java app created with aim to make the biocuration of published pathogenic human variants easier.
+            </p>
+    </div>
+    <div class="card">
+            <h4>BOOM: Bayes OWL Ontology Merging</h4>
```

Comment:
Dear @justaddcoffee, please don't hate me, but after saying yay to this bit, I'm now having second thoughts about k-boom. So, this being a semi-automatic method that was initially used to construct Mondo, makes me feel like we are 'too down in the weeds' for what seems to be a more generic 'Monarch Tools' page. 
If we're doing this, should we also be talking about ODK? 
..... I may be splitting hairs. What do you think, Justin? 
And, @cmungall - should we keep or axe for now?

Reply:
Okay with me to remove k-boom